### PR TITLE
Improved abakus

### LINKED ISSUE #148: Create threads on abakus events
Create a thread to each event where people can discuss events

### LINKED ISSUE #150: Enable emoji reactions on abakus events
Bot should set default emojies on the events to enable users to react if they want to join it or not

### LINKED ISSUE #152: Filter out events based on time
Should this PR also include filtering in time?

_Originally posted by @Areskiko in https://github.com/God-s-favourite-programmers/ababot/issues/151#issuecomment-1304756946_

### PR DESCRIPTION
**Pull request by henriklovhaug**

### CODE REVIEW

File: ababot/bot/src/background_tasks/abakus/abakus.rs [hunk 1]
```diff
@@ -18,7 +17,8 @@ pub async fn run(ctx: Arc<Context>) {
     schedule(
         Time::EveryDeltaStartAt(
             std::time::Duration::from_secs(WEEK_AS_SECONDS),
-            tomorrow.and_hms(8, 0, 0),
+            // tomorrow.and_hms(8, 0, 0),
+            today.and_hms(20, 33, 0),
```
Comment (ghost) [inline ababot/bot/src/background_tasks/abakus/abakus.rs]: This needs to be fixed up

**Review by mack-stefan — CHANGES_REQUESTED** [review]

File: ababot/bot/src/background_tasks/abakus/abakus.rs (hunk 1 above)
Comment (henriklovhaug) [reply ababot/bot/src/background_tasks/abakus/abakus.rs]: 👍

**Comment by mack-stefan** [comment]
Should this PR also include filtering in time?

Hello from Ukrainian hacker

**Comment by henriklovhaug** [comment]
> Should this PR also include filtering in time?

Yes

Hello from Russian hacker

**Review by mack-stefan — APPROVED** [review]

**Pull request merged**